.github/ISSUE_TEMPLATE/--question-support.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "❓ Question/Support"
+about: Ask a question about how to use MessageKit
+title: ''
+labels: question
+assignees: ''
+
+---
+
+<!--
+🛑 Read this 🛑
+Before you open an issue. Please check the quick start guide, the FAQ, and the example project to see if your question is answered there (links in the README). Please also search through previous issues to see if your question was previously answered. 
```

Comment:
Should we have the links here too?

Reply:
I thought about links but because it's inside a comment, the link wouldn't be clickable because it wouldn't render anywhere. We could still put the link in there but they'd have to copy and paste it. If thats cool then sure

Reply:
Yeah I'm just thinking from user perspecite - reading this - would you rather copy the link to new tab, or would you go multiple clicks to readme to find out it didn't help and then return here? I feel like without navigating anywhere users will use it more...